CRAN_questions.md

```diff
@@ -0,0 +1,19 @@
+This document briefly presents some questions this working group has discussed about CRAN to help it being successful.
+
+ * CRAN does an amazing amount of work. What are the most critical issues that CRAN is facing?
+ * CRAN has worked relatively similar for the first 20 years. What is the future of CRAN for the next 20 years?
+ * There is some tension between packages' developers and users' interests respect to checks. What is the purpose of CRAN checks? 
+ * What other are important aspects people should have in mind when volunteering to help CRAN?
+ * Currently there are two repositories with agreements to avoid name conflicts, CRAN and Bioconductor. Would CRAN support the development of additional repositories along the lines of Bioconductor? What kinds of policies would that involve?
+ * The CRAN team is very few people volunteering and lots of pressure. How could people and outside groups like us help?
+
+Related to the last point this working group have been discussing several approaches:
```

Comment:
I would recommend deferring these items until after our initial engagement, i.e., after the questions above have been answered.

Reply:
I agree that we should start from a position of listening to CRAN, but I think there's still some benefit to proposing a couple of concrete actions that we could start with.

Reply:
I think it is important to show willingness to work with them (or under their direction) instead of just talking about CRAN. But let's see what is the consensus about this and what, if any, proposals those could be. There is also the danger of derailing on those actions too.